python/convert.py

```diff
@@ -0,0 +1,7 @@
+import mgp
+from json import loads
+
+@mgp.function
+def str2object(string: str) -> mgp.Any:
+    if string:
+        return loads(string)
```

Comment:
What do you return otherwise? Do we need to throw error?

Reply:
No no, this is not a check if it is a string, that is automatic since in function definition we have str, this checks if the input string is empty, because if it is json loads gets crazy